Rubberduck.sln

```diff
@@ -138,6 +145,26 @@ Global
 		{AC4F1D22-D74B-45FF-AB0C-CC2A104FE023}.Release|x64.Build.0 = Release|Any CPU
 		{AC4F1D22-D74B-45FF-AB0C-CC2A104FE023}.Release|x86.ActiveCfg = Release|Any CPU
 		{AC4F1D22-D74B-45FF-AB0C-CC2A104FE023}.Release|x86.Build.0 = Release|Any CPU
+		{4B9BD3FE-DDC8-4842-BC3D-B8EF43011F0C}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
+		{4B9BD3FE-DDC8-4842-BC3D-B8EF43011F0C}.Debug|Any CPU.Build.0 = Debug|Any CPU
```

Comment:
How many configurations do we need?

Reply:
Evidently not enough. I didn't put it there, at least not explicitly. I'll find out where it came and why.